packages/e2e-test-app/windows/ExperimentalFeatures.props

```diff
@@ -3,6 +3,8 @@
 
   <PropertyGroup Label="Microsoft.ReactNative Experimental Features">
     <UseHermes>false</UseHermes>
+    
+    <ExperimentalFeaturesSet>true</ExperimentalFeaturesSet>
```

Comment:
>ExperimentalFeaturesSet [](http://example.com/codeflow?start=5&length=23)

Could we prefix it with "React"?
All these properties are global. We better to have our project-specific prefixes for all our properties. This is considered to be a good practice for MSBuild code. E.g.C++/WinRT code uses the CppWinRT prefix and their properties look like CppWinRTNamespaceMergeDepth, etc.